src/thunderbots/software/ai/navigator/obstacle/placeholder_obstacle.cpp

```diff
@@ -0,0 +1,21 @@
+#include "ai/navigator/obstacle/placeholder_obstacle.h"
+
+Polygon PlaceholderObstacle::getBoundaryPolygon(const Robot& robot,
+                                                double robotRadiusScaling,
+                                                double velocityProjectionScaling)
+{
+    (void)robot;
+    (void)velocityProjectionScaling;
+    static double ROOT_THREE_BY_TWO =
+        0.86602540378;  // static, so it's not recalculated 4 times per call
+    double radius = ROBOT_MAX_RADIUS_METERS * robotRadiusScaling;
+
+    return Polygon(std::vector<Point>({
```

Comment:
nit: initializing this with an `std::initializer_list` like this: 
`Polygon{Point(0, 0), Point(1, 0), Point(1,1)}` will perform better by avoiding heap allocations.

Reply:
OK